core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -327,10 +327,10 @@ class ControllerBrokerRequestBatch(controller: KafkaController) extends  Logging
         controllerContext.partitionLeadershipInfo.keySet
       else
         partitions
-      if (controller.deleteTopicManager.partitionsToBeDeleted.isEmpty)
+      if (controller.deleteTopicManager.getPartitionsToBeDeleted().isEmpty)
```

Comment:
since this method acts as a value (i.e. no side effects), we typically don't use ().
see the call to "isEmpty" immediately after.